### ISSUE TITLE
Can't edit works imported with a location

### ISSUE THREAD

**Issue by mark-dce** [curationexperts-deprecated/tenejo-v1#135]
**ISSUE**
When I import records using the importer, they show up as expected, but I get a "We're sorry, but something went wrong error when I try to edit them."

**STEPS TO REPRODUCE**
1. Download the sample file in this gist [loc-images.csv](
https://gist.github.com/mark-dce/9c11624399c3983cac2c9dd867560b28#file-loc-images-csv) to use for the test
1. Login as a user with admin privileges
1. Create a new empty collection for the test
1. Start a new CSV import
1. Select the file `loc-images.csv` (from step 1)
1. Select the collection you created in step 3
1. Begin the import
1. Wait for the import to finish
1. View the collection
1. Edit one of the three items with a location specified ("In March read the books you've always meant to read", "See America Welcome to Montana / / R. Halls","Prints for the people A national exhibition...")
1. Click on `Edit` from that image's single item show page

**EXPECTED RESULTS**
I should be able to edit the metadata for the imported image.

**ACTUAL RESULTS**
I get the "We're sorry, but something went wrong" page.
see https://app.honeybadger.io/projects/59580/faults/45325894

NOTE: I've tried providing both string "Montana" and URI http://www.geonames.org/5667009/montana.html values for the location field.

**Comment by bess** [comment]
Hyrax expects to receive a string like `http://sws.geonames.org/4896861/` (that's another vesion of the Illinios geonames URI you were trying to attach, @mark-dce )  and then transform it into a `Hyrax::ControlledVocabularies::Location` object, like this one:
```
[#<Hyrax::ControlledVocabularies::Location:0x56faa18 ID:<http://sws.geonames.org/4896861/>>]
```

@mark-dce How do you *want* to provide the location? Is it okay if we ask for a geonames URI, even if we have to munge it to get it into the right format?

**Comment by mark-dce** [comment]
I'm fine with requiring a valid genomes URI for the first pass.  That's what I thought I was providing in the spreadsheet I linked to - 
https://gist.github.com/mark-dce/9c11624399c3983cac2c9dd867560b28#file-loc-images-csv

I'm using things like `http://www.geonames.org/4896861/illinois.html` and still get the error.  I don't know how to look up the `sws.geonames` version of the URI, so you'll have to teach me how to do that if that's what's required.  

Can you make a copy of the csv linked to this ticket and update it with the correctly formatted entries in the `location` field and then let Rachel & I know what we need to do?

**Comment by bess** [comment]
Once this is merged you should be able to include geonames urls like `http://www.geonames.org/6252001/united-states.html` in the `location` field of any CSV. Those URLs should then get resolved to their GeoNames label in the UI, and the record should be editable after creation.

**Comment by mark-dce** [comment]
When I have valid genomes URIs in my CSV manifest, I can edit the works after they're imported.  It looks like there's a bug in Hyrax and it's displaying the URIs instead of the labels, but that's upstream and I'll create a separate ticket for that.

![135 - editing location from csv import](https://user-images.githubusercontent.com/3064318/52670905-62b31900-2edf-11e9-8b52-54c87d012f68.png)
